scilpy/tracking/utils.py

```diff
@@ -79,12 +79,13 @@ def add_out_options(p):
 
 
 def verify_streamline_length_options(parser, args):
-    if not args.min_length > 0:
-        parser.error('min_length must be > 0, {}mm was provided.'
+    if not args.min_length >= 0:
```

Comment:
Do we enforce minimum number of points of 2 somewhere? (Less than 2 points is hardly a streamline.)

Reply:
No, not yet. But the default parameter is 10 mm. So 1-point streamlines would only be accepted if the user manually specifies --min_length 0 (which can be useful when debugging something. You can even verify the number of streamlines of only one point that were created after).

Reply:
I'm just worried about the saving/loading of streamlines with 0 point (empty list) or 1 point. Do we know if our tools support these? If not, should we include some sort of warning when `min_length / step_size < 1`?

Reply:
oh that's right! Ok it might not matter. It's the user's problem in the end. If he does want it to happen he should set `min_length = step_size`.